src/logic/Calculate.js

```diff
@@ -0,0 +1,40 @@
+import Operate from './Operate';
+
+const Calculate = (data, buttonName) => {
+  let { total, next, operation } = data;
```

Comment:
this function is all lower-case spelled in requirement [[see requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/onlyCalculate.png)]

this is not obligatory, it's just good practice so that we can easily have Component names in CamelCase, and we have no confusion between component names and function names